packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -298,7 +298,9 @@ function focusElement(element: HTMLElement | null, scroll = false) {
     }
   } else if (element != null) {
     try {
-      element.focus();
+      requestAnimationFrame(() => {
+        element.focus();
+      });
```

Comment:
I tried adding this raf to the `.focus` calls within `focusWithoutScrolling` in an effort to remove it from https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/focus/src/FocusScope.tsx#L382, but it kept breaking a bunch of tests. Settled with this change as a quick fix for release